server.js

```diff
@@ -1,15 +1,62 @@
+require('newrelic');
 const express = require('express');
 const morgan = require('morgan');
 const path = require('path');
 const cors = require('cors');
+// const requestPromise = require('request-promise');
 
 const app = express();
-const port = process.env.PORT || 3000;
+const port = process.env.PORT || 7070;
+
+// bundles for each react component
+// const bundles = require('../public/services')
+const clientBundles = './public/services';
+// bundles for each servers
+const serverBundles = './templates/services';
+// local network address for each service
+const serviceConfig = require('./service-config.json');
+// fetch each bundle files and write new files into local directory
+const services = require('./loader.js')(clientBundles, serverBundles, serviceConfig);
```

Comment:
These comments aren't currently accurate: clientBundles and serverBundles are strings, serviceConfig doesn't have local network addresses, and the comment for services doesn't mention that it actually stores the component functions.